coc.md

```diff
@@ -18,6 +18,20 @@ layout: page
 		<p>Some LGBTQ in Tech spaces may have additional rules in place, which will be made clearly available to participants. Participants are responsible for knowing and abiding by these rules.</p>
 
 		<div class="hr"></div>
+
+		<a name="conduct"></a>
+		<h2>Conduct</h2>
```

Comment:
I'd like to see this as an additional item under Harassment rather than a whole separate section. As the PR stands now it seems like it would send the message that ban campaigns are a HUGE issue on this slack, which isn't true.

Reply:
I would actually like to rework the section to de-emphasize harassment and talk about other kinds of conduct. Lumping it all under harassment I think directs things the wrong way, but this was an incremental change.

Reply:
If it were me I'd hesitate to make a new _Conduct_ section until we'd had a larger values/vision conversation about it.